Pull request

larowlan opened the pull request:
Hey
Are you interested in combining work on this.
I've got this completely rewritten to use the core apis and drupal.org/project/user_files to manage the uploads and permissions.
This means the files go via the private file system and everything is pretty sweet.
Also I've got access to create full projects on Drupal.org and would be keen to put this up there. Are you interested in being a co-maintainer?

Lee

Commit:
Reorder/structure the files into more logical fashion, move admin functions and menu callbacks to an inc file, added doc blocks etc to functions, split long form_alter hook into multiple hook_form_FORM_ID_alter hooks

Commit:
More work towards core api usage for files (note broken - WIP)

Commit:
First working version

Drupalfarms commented:
Hi Lee,

Excellent initiative..

Yes, I am totally in favour of cooperation.
No, a this moment I neither have time nor active requirements.

How can I be of help?
Maybe I could do testing?

Best, Lex

On Wed, Oct 12, 2011 at 1:36 PM, Lee Rowlands
reply@reply.github.com
wrote:

> Hey
> Are you interested in combining work on this.
> I've got this completely rewritten to use the core apis and drupal.org/project/user_files to manage the uploads and permissions.
> This means the files go via the private file system and everything is pretty sweet.
> Also I've got access to create full projects on Drupal.org and would be keen to put this up there. Are you interested in being a co-maintainer?
> 
> Lee
> 
> You can merge this Pull Request by running:
> 
>  git pull https://github.com/larowlan/uc_attribute_files master
> 
> Or you can view, comment on it, or merge it online at:
> 
>  https://github.com/Drupalfarms/uc_attribute_files/pull/8
> 
> -- Commit Summary --
> - Reorder/structure the files into more logical fashion, move admin functions and menu callbacks to an inc file, added doc blocks etc to functions, split long form_alter hook into multiple hook_form_FORM_ID_alter hooks
> - More work towards core api usage for files (note broken - WIP)
> - First working version
> 
> -- File Changes --
> 
> A uc_attribute_files.inc (72)
> M uc_attribute_files.info (5)
> M uc_attribute_files.install (22)
> M uc_attribute_files.module (773)
> 
> -- Patch Links --
> 
>  https://github.com/Drupalfarms/uc_attribute_files/pull/8.patch
>  https://github.com/Drupalfarms/uc_attribute_files/pull/8.diff
> 
> ## 
> 
> Reply to this email directly or view it on GitHub:
> https://github.com/Drupalfarms/uc_attribute_files/pull/8

## 

## 

Lex van Sonderen - lex.van.sonderen@drupalfarms.comhttp://@drupalfarms.com
Phone: +31 6 45608868 - Web: http://www.drupalfarms.com
Drupalfarms - Uw Drupal site voor € 2500 in 2 weken

larowlan commented:
Hi Lex
So you'd be happy if I put this up on d.o?
If so what's your d.o account -> I'll give you commit/edit access.

LR

On Wed, 2011-10-12 at 07:10 -0700, Drupalfarms wrote:

> Hi Lee,
> 
> Excellent initiative..
> 
> Yes, I am totally in favour of cooperation.
> No, a this moment I neither have time nor active requirements.
> 
> How can I be of help?
> Maybe I could do testing?
> 
> Best, Lex
> 
> On Wed, Oct 12, 2011 at 1:36 PM, Lee Rowlands
> reply@reply.github.com
> wrote:
> 
> > Hey
> > Are you interested in combining work on this.
> > I've got this completely rewritten to use the core apis and drupal.org/project/user_files to manage the uploads and permissions.
> > This means the files go via the private file system and everything is pretty sweet.
> > Also I've got access to create full projects on Drupal.org and would be keen to put this up there. Are you interested in being a co-maintainer?
> > 
> > Lee
> > 
> > You can merge this Pull Request by running:
> > 
> >  git pull https://github.com/larowlan/uc_attribute_files master
> > 
> > Or you can view, comment on it, or merge it online at:
> > 
> >  https://github.com/Drupalfarms/uc_attribute_files/pull/8
> > 
> > -- Commit Summary --
> > - Reorder/structure the files into more logical fashion, move admin functions and menu callbacks to an inc file, added doc blocks etc to functions, split long form_alter hook into multiple hook_form_FORM_ID_alter hooks
> > - More work towards core api usage for files (note broken - WIP)
> > - First working version
> > 
> > -- File Changes --
> > 
> > A uc_attribute_files.inc (72)
> > M uc_attribute_files.info (5)
> > M uc_attribute_files.install (22)
> > M uc_attribute_files.module (773)
> > 
> > -- Patch Links --
> > 
> >  https://github.com/Drupalfarms/uc_attribute_files/pull/8.patch
> >  https://github.com/Drupalfarms/uc_attribute_files/pull/8.diff
> > 
> > ## 
> > 
> > Reply to this email directly or view it on GitHub:
> > https://github.com/Drupalfarms/uc_attribute_files/pull/8
> 
> ## 
> 
> ## 
> 
> Lex van Sonderen - lex.van.sonderen@drupalfarms.comhttp://@drupalfarms.com
> Phone: +31 6 45608868 - Web: http://www.drupalfarms.com
> Drupalfarms - Uw Drupal site voor € 2500 in 2 weken

Drupalfarms commented:
Hi Lee,

Sure go on... I am "Lex van Sonderen" at d.o

Lex

On Fri, Oct 14, 2011 at 1:49 AM, Lee Rowlands
reply@reply.github.com
wrote:

> Hi Lex
> So you'd be happy if I put this up on d.o?
> If so what's your d.o account -> I'll give you commit/edit access.
> 
> LR
> 
> On Wed, 2011-10-12 at 07:10 -0700, Drupalfarms wrote:
> 
> > Hi Lee,
> > 
> > Excellent initiative..
> > 
> > Yes, I am totally in favour of cooperation.
> > No, a this moment I neither have time nor active requirements.
> > 
> > How can I be of help?
> > Maybe I could do testing?
> > 
> > Best, Lex
> > 
> > On Wed, Oct 12, 2011 at 1:36 PM, Lee Rowlands
> > reply@reply.github.com
> > wrote:
> > 
> > > Hey
> > > Are you interested in combining work on this.
> > > I've got this completely rewritten to use the core apis and drupal.org/project/user_files to manage the uploads and permissions.
> > > This means the files go via the private file system and everything is pretty sweet.
> > > Also I've got access to create full projects on Drupal.org and would be keen to put this up there. Are you interested in being a co-maintainer?
> > > 
> > > Lee
> > > 
> > > You can merge this Pull Request by running:
> > > 
> > >  git pull https://github.com/larowlan/uc_attribute_files master
> > > 
> > > Or you can view, comment on it, or merge it online at:
> > > 
> > >  https://github.com/Drupalfarms/uc_attribute_files/pull/8
> > > 
> > > -- Commit Summary --
> > > - Reorder/structure the files into more logical fashion, move admin functions and menu callbacks to an inc file, added doc blocks etc to functions, split long form_alter hook into multiple hook_form_FORM_ID_alter hooks
> > > - More work towards core api usage for files (note broken - WIP)
> > > - First working version
> > > 
> > > -- File Changes --
> > > 
> > > A uc_attribute_files.inc (72)
> > > M uc_attribute_files.info (5)
> > > M uc_attribute_files.install (22)
> > > M uc_attribute_files.module (773)
> > > 
> > > -- Patch Links --
> > > 
> > >  https://github.com/Drupalfarms/uc_attribute_files/pull/8.patch
> > >  https://github.com/Drupalfarms/uc_attribute_files/pull/8.diff
> > > 
> > > ## 
> > > 
> > > Reply to this email directly or view it on GitHub:
> > > https://github.com/Drupalfarms/uc_attribute_files/pull/8
> > 
> > ## 
> > 
> > ## 
> > 
> > Lex van Sonderen - lex.van.sonderen@drupalfarms.comhttp://@drupalfarms.com
> > Phone: +31 6 45608868 - Web: http://www.drupalfarms.com
> > Drupalfarms - Uw Drupal site voor € 2500 in 2 weken
> 
> ## 
> 
> Reply to this email directly or view it on GitHub:
> https://github.com/Drupalfarms/uc_attribute_files/pull/8#issuecomment-2401749

## 

## 

Lex van Sonderen - lex.van.sonderen@drupalfarms.comhttp://@drupalfarms.com
Phone: +31 6 45608868 - Web: http://www.drupalfarms.com
Drupalfarms - Uw Drupal site voor € 2500 in 2 weken

Commit:
Allows form to work with multiple attributes, including two file attributes

Commit:
Fixed issue with option names as file extensions

Commit:
Added support for Pressflow

Commit:
Merge branch 'master' of github.com:larowlan/uc_attribute_files

Commit:
Fixed issue with some attributes not having default value and hence not working in the cart
